config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml

```diff
@@ -38,7 +38,7 @@ presubmits:
     labels:
       preset-dind-enabled: "true"
       preset-kind-volume-mounts: "true"
-      preset-azure-cred: "true"
```

Comment:
are there any other jobs (eg. periodic CAPI e2e) that we're expecting to get logs from that should also be updated?

Reply:
Yes, looks like we should do the same for `pull-cluster-api-provider-azure-capi-e2e`--good thinking. Those are the two places that `./scripts/ci-e2e.sh` is called.

Reply:
Derp, nevermind--we are already there: that was why `preset-azure-cred-only` was added in the first place. So I *think* that covers it.